src/containers/NahmiiHoc/saga.js

```diff
@@ -518,6 +510,10 @@ export function* startChallenge({ stageAmount, currency, options }) {
     let errorMessage = logErrorMsg(e);
     if (errorMessage.match(/gas.*required.*exceeds/i) || errorMessage.match(/out.*of.*gas/i)) {
       errorMessage = getIntl().formatMessage({ id: 'gas_limit_too_low' });
+    } else if (errorMessage.match(/balance.*not.*synchronised/i)) {
```

Comment:
A style comment: Personally I find "stroustrup" style for curly braces easier to read when you have many subsequent if/elseif/else blocks. YMMV

Reply:
Agreed. I feel the same. I will update the eslint config to allow this style.